docker/build-tools/build-and-push.sh

```diff
@@ -40,6 +40,8 @@ fi
 ${CONTAINER_CLI} ${CONTAINER_BUILDER}  --target build_tools --build-arg "ISTIO_TOOLS_SHA=${SHA}" --build-arg "VERSION=${VERSION}" -t "${HUB}/build-tools:${VERSION}" -t "${HUB}/build-tools:${BRANCH}-latest" .
 # shellcheck disable=SC2086
 ${CONTAINER_CLI} ${CONTAINER_BUILDER}  --build-arg "ISTIO_TOOLS_SHA=${SHA}" --build-arg "VERSION=${VERSION}" -t "${HUB}/build-tools-proxy:${VERSION}" -t "${HUB}/build-tools-proxy:${BRANCH}-latest" .
+# shellcheck disable=SC2086
+${CONTAINER_CLI} ${CONTAINER_BUILDER}  --build-arg "ISTIO_TOOLS_SHA=${SHA}" --build-arg "VERSION=${VERSION}" -t "${HUB}/build-tools-centos:${VERSION}" -t "${HUB}/build-tools-cetnos:${BRANCH}-latest" -f Dockerfile.centos .
```

Comment:
typo

Reply:
@howardjohn it seems tide was faster than me on this.

Reply:
Ill send a follow up, good catch. Any other comments? I was hoping to get your feedback on this. This was mostly copy+pasted from the current dockerfile in istio/istio

Reply:
Nope, I'm assuming this Dockerfile works and is able to build current proxy. It's been ages since I've done that work...

Reply:
I cannot confirm its perfect, but it does build a binary that runs. Probably will require one tweak or another, but should get things moving